.changeset/chilly-coats-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/draft-tag": minor
```

Comment:
Do we need a bump for component library as well for the icon being added?

Reply:
Yes, you're right, thanks!